### DISCUSSION TITLE
Grafana 8.0.2 Table column sorting works in a wrong way, anyhelp?

### DISCUSSION THREAD

**Discussion by daecheol93** [grafana/grafana#39820] [Dashboards, Panels & Visualizations]
Hi, 
I made a dashboard of linux server's secure log like this,

![image](https://user-images.githubusercontent.com/82934184/135373864-966305c6-0069-44ac-8ad5-ca8e1685f14d.png)

I have two question.
first, I'm using Grafana 8.0.2 version. when I sort day column, it doesn't work well. so I googled it and I modified Panel JSON. here it is.
![image](https://user-images.githubusercontent.com/82934184/135374045-733c5aa5-2a58-4fa5-ab58-0fa9ffce9060.png)

I only put it "type": "number" in there. but it still doesn't work. anybody knows why it is?

secondly, I have no grafana-ui packages on my grafana server.
Do I have to install them? please give an advice. I'm a rookie. 
Thx, I hope you have a nice day :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
